libs/x-content/src/test/java/org/elasticsearch/common/xcontent/ObjectParserTests.java

```diff
@@ -203,7 +205,7 @@ public void setTest(int test) {
         {
             XContentParser parser = createParser(JsonXContent.jsonXContent, "{\"not_supported_field\" : \"foo\"}");
             XContentParseException ex = expectThrows(XContentParseException.class, () -> objectParser.parse(parser, s, null));
-            assertEquals(ex.getMessage(), "[1:2] [the_parser] unknown field [not_supported_field], parser not found");
+            assertEquals(ex.getMessage(), "[1:26] [the_parser] unknown field [not_supported_field], parser not found");
```

Comment:
This change is because the parser now errors out *after* it has read the unknown field name, rather than before.

Reply:
I reverted this, was easy enough to preserve the old behaviour which makes more sense anyway